packages/mux-player/src/index.ts

```diff
@@ -217,6 +218,24 @@ class MuxPlayerInternal {
     const supportsVolume = await hasVolumeSupportAsync();
     this._setState({ supportsVolume });
   }
+
+  async _setUpPlayerReady(el: MuxPlayerElement) {
+    const { video } = el;
+    if (!video) {
+      console.error("The video element is missing!");
+      return;
+    }
+
+    await Promise.all([
+      // If there is no video src on construction don't be waiting on loadstart.
+      // It might look like we need to check video.networkState here also but
+      // the loadstart event is always fired a tick after the video src is set.
+      video.src && promisify(video.addEventListener, video)("loadstart"),
```

Comment:
nit (non-blocking): We could simplify `promisify` by removing the concept of a `ctx` and simply binding here, e.g.:
```js
video.src && promisify(video.addEventListener.bind(video))("loadstart"),
```
Defer to you on preference.